src/components/header/createHeader.ts

```diff
@@ -19,4 +19,25 @@ const createHeader = (): HTMLElement => {
   return mainHeader;
 };
 
+export const createSelectionStatus = (): void => {
+  const heading = document.querySelector("h1");
```

Comment:
Lo que estáis buscando no es un encabezado, es el título de la página.

Reply:
Pero el h1 es un heading no?

Reply:
Y un 23 es un número, pero no lo llamamos `number`, lo llamamos como lo que representa.